deployment/ansible/roles/bootstrap/tasks/setup-service.yml

```diff
@@ -67,6 +79,22 @@
     - {url: "{{ dcld_download_url }}", dest: "{{ dcld.path }}", checksum: "{{ dcld_binary_checksum }}"}
     - {url: "{{ cosmovisor_download_url }}", dest: "{{ cosmovisor.path }}", checksum: "{{ cosmovisor_binary_checksum }}"}
 
+- name: copy script files to home directory
```

Comment:
please adjust [manual](https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/master/docs/running-node-manual/genesis-vn.md#10-run-node) deployment docs related to that change